hudi-client/src/test/java/org/apache/hudi/io/TestHoodieCommitArchiveLog.java

```diff
@@ -397,10 +402,81 @@ public void testArchiveCommitCompactionNoHole() throws IOException {
         timeline.containsInstant(new HoodieInstant(false, HoodieTimeline.COMMIT_ACTION, "107")));
   }
 
+  @Test
+  public void testArchiveCommitAndDeepCopy() throws IOException {
```

Comment:
Is this method used to test the enum issue in Avro? If so, you can remove it.

Reply:
> Is this method used to test the enum issue in Avro? If so, you can remove it.

Yes, be removed.